qdrant/v1.2.x/quantization.md

```diff
@@ -46,8 +46,19 @@ Please refer to the [Quantization Tips](#quantization-tips) section for more inf
 
 ## Product Quantization
 
-Currently, Work-in-progress.
+*Available in Qdrant since v1.2.0*
 
+Product quantization is a method of compressing vectors to minimize their memory usage by dividing them into 
```

Comment:
Instead of directly diving into the technical stuff here, I think it would be useful to add a single sentence that broadly describes what this is.

For scalar quantization we have this:

> Scalar quantization, in the context of vector search engines, is a compression technique that compresses vectors by reducing the number of bits used to represent each vector component.

Maybe we can add something like this:

> Product quantization, in the context of vector search engines, is a compression technique that compresses vectors by reducing the number of dimensions by grouping a number of elements in buckets, to reduce the conditionality when searching.

(this is definitely wrong, so please correct it :) )